quickstarts/fullstack/next.mdx

```diff
@@ -15,57 +15,115 @@ description: Learn how to quickly add authentication and user profile in your Ne
 ></iframe>
 </Frame>
 
-## Install `@teamhanko/hanko-elements`
+## Create a Next.js application
+Run the following command to [create a new Next.js application](https://nextjs.org/docs/app/getting-started/installation):
 
-Once you've initialized your Next app, installing hanko-elements provides you with access to the prebuilt components: `hanko-auth` and `hanko-profile`.
+<Note>
+  During the Next.js setup you could pick between two routers, the [App Router](https://nextjs.org/docs/app) and the [Pages Router](https://nextjs.org/docs/pages).
+  Ensure you add the correct code and follow the correct directory structure for Router that you picked.
+</Note>
 
-<CodeGroup>
+<CodeGroup> 
+```bash terminal
+npx create-next-app@latest project-name
+```
+</CodeGroup>
+
+
+
+## Install `@teamhanko/hanko-elements`
+Once you've initialized your NextJS app, installing hanko-elements provides you with access to the prebuilt components: `hanko-auth` and `hanko-profile`.
+
+<CodeGroup> 
 ```bash npm
+cd project-name
 npm install @teamhanko/hanko-elements
 ```
 
 ```bash pnpm
+cd project-name
 pnpm add @teamhanko/hanko-elements
 ```
 
 ```bash bun
+cd project-name
 bun add @teamhanko/hanko-elements
 ```
 
 ```bash yarn
+cd project-name
 yarn add @teamhanko/hanko-elements
 ```
 </CodeGroup>
 
-## Add the Hanko API URL
 
-Retrieve the API URL from the [Hanko console](https://cloud.hanko.io/) and place it in your `.env` file.
+## Setup your Hanko project
+Go to the [Hanko console](https://cloud.hanko.io/) and [create a project for this application.](/setup-hanko-cloud)
+
+<Note>
+  During creation make sure to input the URL you will be developing on as the `APP URL`.\
+  (Most likely http://localhost:3000/) \
+ \
+  If you already have a project you can change the URL at:\
+  `'your project' > Settings > URLs > App URL`.
```

Comment:
I don't think that we need this, since this is focused on new users and they most likely don't have a project yet.